### DISCUSSION TITLE
Can anyone extract links from 2ix2 which has a lot of German channels

### DISCUSSION THREAD

**Discussion by SpongeBu** [iptv-org/community#306] [Q&A]
I was wondering if anyone can extract links from 2ix2 because that would be nice

**Comment by Sphinxroot** [comment]
2ix2 Have Token

https://www.livestreamde.com/
those can be used  outside of site

#EXTVLCOPT:http-referrer=https://www.livestreamde.com/
https://stream.hdtvizlecanli.com/........ .m3u8

or Kodi  Playlistloader
.m3u8|Referer=https://www.livestreamde.com/

But manny channel have dmca  blocked in here , so they wont be in the list

**Comment by mark-baumann** [comment]
is it technical possible to extract the data to a server and then stream it from an IPTV Source?

**Reply by BellezaEmporium** [reply]
That is basically what panels like Xtream-Codes do.

**Reply by mark-baumann** [reply]
yes but this are all ready solutions. And many don´t work with Kodi. I tried a lot of repos, but get some errors

**Reply by mark-baumann** [reply]
all addons from kodinerds dont work at kodi version 20?????

**Reply by BellezaEmporium** [reply]
You might need to ask @Sphinxroot about it, but normally everything works.

**Comment by Sphinxroot** [comment]
https://repo.kodinerds.net/  if not updated to py v3  it wont work anymore  .. but if you are on pc   u can use  leia in portable mode  but  deactivate  version check  and  addons update  . you will have to updated them manualy in the addon folder ,   playlistloader  have py2 and py3  so link will work on both version  using m3u list in it
